pkg/volume/csi/csi_attacher_test.go

```diff
@@ -940,10 +928,12 @@ func newTestWatchPlugin(t *testing.T, csiClient *fakecsi.Clientset) (*csiPlugin,
 		t.Fatalf("cannot assert plugin to be type csiPlugin")
 	}
 
-	for {
-		// Wait until the informer in CSI volume plugin has all CSIDrivers.
-		if csiPlug.csiDriverInformer.Informer().HasSynced() {
-			break
+	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
+		for {
```

Comment:
Probably should have a timeout after which tests fails and not wait indefinitely?